services/workflow/workflow-manager/src/main/java/com/milesight/beaveriot/rule/manager/po/WorkflowPO.java

```diff
@@ -0,0 +1,54 @@
+package com.milesight.beaveriot.rule.manager.po;
+
+import jakarta.persistence.*;
+import lombok.Data;
+import lombok.experimental.FieldNameConstants;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+@Data
+@Entity
+@FieldNameConstants
+@Table(name = "t_flow")
+@EntityListeners(AuditingEntityListener.class)
+public class WorkflowPO {
+    @Id
+    @Column(name = "id")
+    private Long id;
+
+    @Column(name = "version")
+    private Integer version = 1;
```

Comment:
JPA already has @Version annotations, considering whether it can be implemented directly using annotations

Reply:
I have used `@Version` but it cannot prevent users from overwritten workflow data between different requests. So I keep the judgement code.